### DISCUSSION TITLE
Cannot resolve symbol 'DgsConstants'

### DISCUSSION THREAD

**Discussion by Matt-Cam** [Netflix/dgs-framework#259] [Q&A] [answered]
Hi! 
I am going through the DGS getting started docs. The tutorial has some code snippets that reference ```DgsConstants```. From my understanding,  ```DgsConstants``` will be auto-generated on build, but is there a way to access ```DgsConstants``` during development before building the codebase? Intellij is showing errors because it doesn't know about build results before building. 

Thanks!
![image](https://user-images.githubusercontent.com/11476729/114628624-24fbd780-9c85-11eb-9ce3-a4f24606dfec.png)

**Comment by paulbakker** [comment, marked as answer]
What you typically want to do after a schema change (or initially when creating a schema), is kick-off code generation:

```
./gradlew generateJava
```

The classes will be generated and recognized by Intellij after that.

**Reply by Matt-Cam** [reply]
Thanks @paulbakker! Any idea what this looks like if and when using Maven instead of Gradle?

**Reply by paulbakker** [reply]
The Maven plugin is maintained by a community member, I'm not really familiar with it. But looking at the docs, it looks like it registers a `generate` task that you can execute: https://github.com/deweyjose/graphqlcodegen#usage
